src/go/rpk/pkg/cli/cmd/redpanda/admin/admin.go

```diff
@@ -0,0 +1,71 @@
+// Package admin provides a cobra command for the redpanda admin listener.
+package admin
+
+import (
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/cli/cmd/common"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/cli/cmd/redpanda/admin/brokers"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/config"
+)
+
+// Command returns the redpanda admin command.
+func Command(fs afero.Fs, mgr config.Manager) *cobra.Command {
```

Comment:
The existing convention is to name functions that create commands `NewXCommand`. Can you rename the new functions in this commit to fit that convention please? This helps with codebase-wide searches and prevents variance in naming, like this commit's `listCommand` vs `Command` vs `decommissionBroker`.

Reply:
`admin.NewAdminCommand` is a bit of stutter, which is recommended to be avoided as per the Go [guidelines](https://blog.golang.org/package-names). I can see this changing to `NewCommand`. I think the current `NewXCommand` is likely due to all of the commands being stuffed in the same package (cmd).

The `listCommand` vs `Command` is a part of the effort to have cleaner public/private hygiene, as per the rpk command planning document. We can keep the list command private, as it does not need to be public. This also sets the standard for future commands that list, in that they can just use the same `listCommand` function.

A very short term goal is to refactor the `NewXCommand` / public / private bits so that they are more cleanly separated and address the public/private hygiene component. This PR is a step towards that direction, and I think it'd be a step backwards to have consistency with the past solely for the sake of consistency -- and it would also mean we cannot have piecemeal refactoring.

Reply:
I've renamed to NewCommand. I've kept the names of the private functions, at most they could be `newListCommand`, etc., as their private-ness makes for a different standard, and four `new`s in a row does not really look too great:

```go
                newListCommand(closures),
                newDescribeCommand(closures),
                newDecommissionBroker(closures),
                newRecommissionBroker(closures),
```